static/js/settings_org.js

```diff
@@ -450,7 +473,10 @@ function update_dependent_subsettings(property_name) {
     if (property_name === 'realm_create_stream_permission' || property_name === 'realm_waiting_period_threshold') {
         set_create_stream_permission_dropdown();
     } else if (property_name === 'realm_video_chat_provider' ||
-               property_name === 'realm_google_hangouts_domain') {
+               property_name === 'realm_google_hangouts_domain' ||
+               property_name === 'realm_zoom_user_id' ||
```

Comment:
`property_name.startsWith('realm_zoom')` could be better?

Reply:
yeah, I like that.